src/gui/kblight.cpp

```diff
@@ -472,7 +472,7 @@ void KbLight::load(CkbSettingsBase& settings){
         }
     }
     emit didLoad();
-    map(currentMap);
+    map(currentMap.count() == 0 ? _map : currentMap);
```

Comment:
Just guessing here, but perhaps there's a problem where if currentMap.count returns zero, map(currentMap) causes a crash, so map(_map) is the safe thing to do.

Likely this scenario isn't possible while the GUI is running, but it might be possible when run as a service prior to login and/or run by a user without a configured profile as yet.